examples/chrono/chrono.cpp

```diff
@@ -0,0 +1,58 @@
+﻿/*
+ * Copyright (c) 2022, Christopher Kormanyos
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <chrono>
+
+#include <avr/io.h>
+
+auto main() -> int
+{
+  // Set the port pin value to low.
+  PORTB |= ~(1U << PORTB5);
+
+  // Set the port pin direction to output.
+  DDRB |= (1U << DDB5);
+
+  auto start = std::chrono::steady_clock::now();
+
+  for(volatile unsigned long toggles = 0UL; toggles < 10000UL; ++toggles)
+  {
+    // Toggle the LED via direct memory access.
```

Comment:
Direct Memory Access (DMA) is a completely different thing than Memory Mapped Input/Output (MMIO) register access.

Reply:
I can change the comment.